static/sass/_pattern_footer.scss

```diff
@@ -1,268 +1,302 @@
 // Footer pattern styling
 
 @mixin footer-title {
-  color: $color-mid-dark;
-  cursor: pointer;
-  line-height: 1.5;
-  margin-bottom: 0;
+    color: $color-mid-dark;
```

Comment:
Sorry... but it should be 2 spaces per indent, not 4!

Reply:
Hi, @deadlight no worries, I have reduced the indents to 2 spaces now. Cheers